yapapi/runner/__init__.py

```diff
@@ -351,13 +382,17 @@ async def fill_work_q():
         except Exception as e:
             print("fail=", e)
         finally:
+            payment_closing = True
             for worker_task in workers:
                 worker_task.cancel()
+            print(agreements_to_pay)
```

Comment:
needs some context info. Plain ids (uuids) will have no meaing for average logs reader.